section3/reflection.md

```diff
@@ -2,12 +2,26 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+It was tough at first, but I recalled much from the growth mindset podcast video and used what I learned there to help. I observed my stress but didn't panic, and used google/youtube as a resource to find the answers. I also posted in slack a couple times and found help from classmates. With a little time and effort, I've been able to figure things out.
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
+Youtube saves videos in an array as a playlist. For example, the array is "videos saved for later", and the items in the array are the specific videos you choose.
+
 3. In your own words, explain what an Array method is.
 
+commands that make changes to a given array. Most common uses are adding/deleting a value from the front/back of an array.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
 
+Loops save us time by running a process as many times as needed until complete automatically instead of manually one instance at a time.
+
+You would use a loop with an array when you want to apply a condition to a set of values all at one time. By grouping the values of the array together and applying the condition to the array, you save time and minimize errors.
+
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+When you ride a carousel, each ride will go around the circle 10 times. Each time you pass by the beginning, a little box keeps track of how many times around that is. When that box sees you've gone around 10 times, it stops the ride so other people can go. Thats a loop.
+
 6. What questions do you still have about Arrays and loops?
+
+I'd love to see some of these concepts applied in real-life applications. I'm sure there is additional context to learn before any of that makes sense, but I'm itching to see how developers use the concepts in everyday work.
```

Comment:
You will get a lot of practice with loops in larger applications in the full time program.